tests/transformers/models/test_causal_lm_models.py

```diff
@@ -5,25 +5,35 @@
 #
 # -----------------------------------------------------------------------------
 
-import json
-
 import pytest
 from transformers import AutoConfig, AutoModelForCausalLM
 
 from QEfficient.utils.constants import Constants
 from QEfficient.utils.device_utils import get_available_device_id
 from tests.utils import get_cloud_ai_100_tokens, set_up
 
-TEST_CONFIG_FILE_PATH = "tests/config.json"
+test_params = [
```

Comment:
We are introducing num_layers as parameter in the config file. This is to fix pytorch vs onnx value mismatch in single layer phi model.

Reply:
- Main reason is because there is no reason for it to be in a separate file - It is not used anywhere else than here.
- Simplifying the configuration for this test. It would be best if the params for this test resides in this file, no need to hunt for another file somewhere in the repo.

Did we debug why single layer model is failing for phi model? We can have mismatch between ONNX and AIC, but PyTorch vs ONNX shouldn't have an issue.

Reply:
yes Pytorch and ONNX outputs are matching, AIC output is different.
![image](https://github.com/user-attachments/assets/b4939497-6c91-40cd-a177-16c1f08196cc)